package.json

```diff
@@ -26,6 +33,7 @@
     "gulp-streamify": "^1.0.2",
     "gulp-stylus": "^2.1.0",
     "gulp-uglify": "^1.5.1",
+    "react": "^0.14.7",
```

Comment:
As near as I can tell, this is the model that React-based libraries seem to be settling on -- declare a peer-dependency to acknowledge the run-time requirement and a dev-dependency to meet the requirement during development. React-DnD, React-Motion, and React-SimpleTabs all follow that model. Now that React-Motion and React-SimpleTabs are dependencies of GeniBlocks, if we don't have the dev-dependency, then on `npm install` we get
```
npm WARN EPEERINVALID react-simpletabs@0.7.0 requires a peer of react@0.12.x - 1.x but none was installed.
npm WARN EPEERINVALID react-motion@0.4.2 requires a peer of react@>=0.13.2 || ^0.14 but none was installed.
```
Peer dependencies don't cause React to be pulled into the bundle, however, so at runtime it is still the case that React is provided by the caller. The difference now is that the GeniBlocks code can/must `require` or `import` React explicitly rather than assuming that it's available as a global and `exposify` is used to make the global React available via `require` or `import`, thus decoupling the GeniBlocks code from the way in which React is provided for it. Moving forward, React will be a more explicit dev-dependency in the upcoming unit test harness PR.